### ISSUE TITLE
Data directory setting crash

### ISSUE THREAD

**Issue by beefeater7** [axcore/tartube#6]
![data directory](https://user-images.githubusercontent.com/10468970/60340968-9e391600-999c-11e9-9feb-5f3df873fd14.PNG)
Clicking [Change] will kill the process.

**Comment by axcore** [comment]
Fixed in v0.4.0

**Comment by m4gnet98** [comment]
Not sure this is fixed, not on windows at least?

```
C:\Users\Andreas\AppData\Local\Tartube\msys64\home\user\tartube>..\..\..\mingw64\bin\python3.exe tartube.py

(python3.exe:22432): GLib-GIO-ERROR **: 01:19:26.145: No GSettings schemas are installed on the system

C:\Users\Andreas\AppData\Local\Tartube\msys64\home\user\tartube>PAUSE
Press any key to continue . . .
```

https://puu.sh/DMAXX/2ee6684965.mp4

**Comment by axcore** [comment]
Fixed (again) in v0.5.0.

**Comment by axcore** [comment]
As of v0.5.0, you can change the location of the data directory, but you can't change to an existing data directory (that already has a tartube.db file in it)

In other words, you can't switch between two or more data directories.

**Comment by axcore** [comment]
As of v0.5.0, you can change the location of the data directory, but you can't change to an existing data directory (that already has a tartube.db file in it)

In other words, you can't switch between two or more data directories.

**Comment by axcore** [comment]
It's a one-line fix; just replace the ../lib/config.py file with this one.

https://github.com/axcore/tartube/blob/master/lib/config.py

**Comment by axcore** [comment]
Fixed in v0.6.0.
